services/catalog/src/simcore_service_catalog/core/events.py

```diff
@@ -23,6 +25,11 @@ async def start_app() -> None:
         if app.state.settings.postgres.enabled:
             await connect_to_db(app)
 
+        # setup connection to director
```

Comment:
I could not select the entire function, but my question is why are you wrapping the handler in  a function?

Reply:
to be honnest I followed @pcrespov lead on that one (same as the connector to the DB). I would say this kind of separate code more. I cannot really say more to that one.